core/src/main/java/hudson/tools/DownloadFromUrlInstaller.java

```diff
@@ -148,6 +146,23 @@ public String getId() {
             if(d==null)     return Collections.emptyList();
             return Arrays.asList(((InstallableList)JSONObject.toBean(d,InstallableList.class)).list);
         }
+        
+        /**
+         * The UI uses this information to populate the drop-down.
+         * 
+         * @return JSON array of installable tools, never null
+         * @since 1.617
+         */
+        public JSONArray getInstallablesJson() throws IOException {
```

Comment:
my understanding is, that `@Restricted` should be used only for deprecated API, which will be removed soon or API which is dangerous to use for any other reason (and IMHO in concordance with [javadoc](https://github.com/kohsuke/access-modifier/blob/master/access-modifier-annotation/src/main/java/org/kohsuke/accmod/Restricted.java#L34)), but if it helps this PR to be merged, I have no problem with it... PR updated